### PR TITLE
Add inputs for CheckDMARC and YARA

### PR DESCRIPTION
**Pull request by mhofert-r7**
### Description

Describe the proposed changes:

  - Add example inputs for CheckDMARC and YARA

### Testing

None documentation change only

## PR Requirements

Developers, verify you have completed the following items by checking them off:

### Style

Review the [style guide](https://komand.github.io/python/style.html)

- [ ] For dependencies, pin [OS package](https://komand.github.io/python/style.html#dockerfile) and [Python package](https://komand.github.io/python/style.html#requirements-txt) versions
- [ ] For security, set least privileged account with ``USER nobody`` in the ``Dockerfile`` when possible
- [ ] For size, use the [slim SDK images](https://komand.github.io/python/sdk.html#sdk-versions) when possible: ``komand/python-3-37-slim-plugin`` and ``komand/python-3-37-plugin``
- [ ] For error handling, use of [PluginException](https://komand.github.io/python/error_handling.html#plugin-exceptions) and [ConnectionTestException](https://komand.github.io/python/error_handling.html#connection-exceptions)
- [ ] For logging, use [self.logger](https://komand.github.io/python/sdk.html#logging)
- [ ] For docs, use [changelog style](https://komand.github.io/python/style.html#changelog)
- [ ] For docs, validate markdown with ``make validate`` which calls ``mdl`` to lint ``help.md``

### Functional Checklist
- [ ] Work fully completed
- [ ] Functional
  - [ ] Any new actions/triggers include JSON [test files](https://komand.github.io/python/style.html#tests) in the `tests/` directory created with `icon-plugin run -c sample $action > tests/$action.json`
  - [ ] Tests should all pass unless it's a negative test. Negative tests have a naming convention of `tests/$action_bad.json`
  - [ ] Unsuccessful tests should fail by raising an exception causing the plugin to die and an object should be returned on successful test
  - [ ] Add functioning test results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -T tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -T all --debug --jq` (use PR format at end)
  - [ ] Add functioning run results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -R tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -R all --debug --jq` (use PR format at end)

### Assessment

You must validate your work to reviewers:

1. Run `make validate` and make sure everything passes
2. Run the assessment tool: `icon-plugin run -A -R all -T all`. For single action validation: `icon-plugin run -A -R tests/my_action.json -T tests/my_action.json`
3. Copy (`icon-plugin ... | pbcopy`) and paste the output in **a new post** on this PR.
4. Add UI screenshot of the workflow used for testing
5. Add UI screenshot of the job output used for testing
6. Add UI screenshot of the artifact (See rules in UI Checklist) used for testing

### CODE REVIEW

File: checkdmarc/help.md [hunk 1]
```diff
@@ -214,11 +223,23 @@ This action will check DMARC records against alternate name servers.
...
+|----|----|-------|--------|-----------|----|-------|
+|domain|string|None|True|Domain to check. e.g. fbi.gov, google.com in alternate nameserver|None|rapid7.com|
+|nameservers|[]string|['1.1.1.1', '1.0.0.1']|True|Nameserver to check against. e.g ["1.1.1.1","1.0.0.1"]|None|["1.1.1.1","1.0.0.1"]|
+|timeout|number|6|True|Timeout in seconds for request. Default is 6 seconds|None|6|
+
+Example input:
+
+```
+{
+  "domain": "whitehouse.gov",
```
Comment (jschipp-r7) [inline checkdmarc/help.md:236]: @mhofert-r7 Can we get this to match the example key? Future versions of icon-plugin will just generate from that key.

Comment (jschipp-r7) [inline checkdmarc/help.md:236]:
```suggestion
  "domain": "rapid7.com",
```

File: checkdmarc/help.md [hunk 2]
```diff
@@ -26,10 +26,19 @@ This action is used to check domains.
 
 ##### Input
 
-|Name|Type|Default|Required|Description|Enum|
-|----|----|-------|--------|-----------|----|
-|domain|string|None|True|Domain to check. e.g. fbi.gov, google.com|None|
-|timeout|number|6.0|True|Timeout in seconds for request. Default is 6 seconds|None|
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|domain|string|None|True|Domain to check. e.g. fbi.gov, google.com|None|rapid7.com|
```
Comment (lmilby-r7) [inline checkdmarc/help.md:31]:
```suggestion
|domain|string|None|True|Domain to check|None|rapid7.com|
```

File: checkdmarc/help.md [hunk 3]
```diff
@@ -214,11 +223,199 @@ This action will check DMARC records against alternate name servers.
...
 
-|Name|Type|Default|Required|Description|Enum|
-|----|----|-------|--------|-----------|----|
-|domain|string|None|True|Domain to check. e.g. fbi.gov, google.com in alternate nameserver|None|
-|nameservers|[]string|['1.1.1.1', '1.0.0.1']|True|Nameserver to check against. e.g ["1.1.1.1","1.0.0.1"]|None|
-|timeout|number|6.0|True|Timeout in seconds for request. Default is 6 seconds|None|
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|domain|string|None|True|Domain to check. e.g. fbi.gov, google.com in alternate nameserver|None|rapid7.com|
+|nameservers|[]string|['1.1.1.1', '1.0.0.1']|True|Nameserver to check against. e.g ["1.1.1.1","1.0.0.1"]|None|["1.1.1.1","1.0.0.1"]|
```
Comment (lmilby-r7) [inline checkdmarc/help.md:229]:
```suggestion
|nameservers|[]string|['1.1.1.1', '1.0.0.1']|True|Nameserver to check against|None|["1.1.1.1","1.0.0.1"]|
```

File: checkdmarc/help.md [hunk 4]
```diff
@@ -214,11 +223,199 @@ This action will check DMARC records against alternate name servers.
...
+|timeout|number|6|True|Timeout in seconds for request. Default is 6 seconds|None|6|
+
+Example input:
+
+```
+{
```
Comment (lmilby-r7) [inline checkdmarc/help.md:235]: example input looks like example output

File: virustotal_yara/komand_virustotal_yara/actions/scan_file/action.py [hunk 5]
```diff
@@ -80,11 +80,3 @@ def matches(built_rules, built_files):
 
         # Returning a array of objects
         return {"results": yara_results}
-
-    def test(self):
```
Comment (lmilby-r7) [inline virustotal_yara/komand_virustotal_yara/actions/scan_file/action.py:84]: Add a test connection if you are taking it away in the action.

**Comment by lmilby-r7** [comment]
Remove references of e.g. in descriptions and regenerate.

File: virustotal_yara/komand_virustotal_yara/actions/scan_file/action.py (hunk 5 above)
Comment (mhofert-r7) [reply virustotal_yara/komand_virustotal_yara/actions/scan_file/action.py:84]: Not sure of we want to keep it. There are no connection inputs and all the test did was see if the yara module loads.

File: checkdmarc/help.md [hunk 6]
```diff
@@ -26,10 +26,19 @@ This action is used to check domains.
...
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|domain|string|None|True|Domain to check|None|rapid7.com|
+|timeout|number|6|True|Timeout in seconds for request. Default is 6 seconds|None|6|
+
+Example input:
+
+```
+{
+  "domain": "whitehouse.gov",
```
Comment (jschipp-r7) [inline checkdmarc/help.md:38]:
```suggestion
  "domain": "example.com",
```

**Pull request merged**